statsmodels/tsa/automatic/sarimax.py

```diff
@@ -3,24 +3,92 @@
 import pandas as pd
 import statsmodels.api as sm
 
-def auto_order(endog, criteria='aic', d=0, max_order=(3, 3), **spec=None):
+
+def auto_order(
+                endog, criteria='aic', d=0, max_order=(3, 3),
+                stepwise=False, **spec):
```

Comment:
At some point we need to add a docstring here, giving at least the Parameters and the Return sections. It would also be good to have a Notes section describing the basics of the stepwise approach and giving the citation.

Reply:
I have added a simple docstring with the citation of Hyndman's paper in the Notes section. I'll add more details to it after completing the function.